src/deterministic.jl

```diff
@@ -40,6 +40,32 @@ function Deterministic(
     return Deterministic(name, ta, scaling_factor_multiplier)
 end
 
+"""
+Construct Deterministic from a CSV file. The file must have a column that is the name of the
```

Comment:
The docstring is incorrect.  Can you change it to a brief description of the format?